.travis.yml

```diff
@@ -15,12 +13,18 @@ python:
 
 matrix:
     include:
-        - python: 3.6
+        # Enable 3.7 without globally enabling sudo and dist: xenial for other build jobs
+        - python: 3.7
+          dist: xenial
+        - python: 3.7
           env: STYLE='code'
-        - python: 3.6
+          dist: xenial
+        - python: 3.7
           env: STYLE='docs'
+          dist: xenial
 
 install:
+    - pip install --upgrade pip setuptools
```

Comment:
Why is adding this necessary? Can we revert this line? IMO, we shouldn't manipulate more of the environment than necessary.

Reply:
Removed.